go/vt/vtgate/plugin_mysql_server.go

```diff
@@ -72,6 +72,11 @@ var (
 	mysqlConnWriteTimeout = flag.Duration("mysql_server_write_timeout", 0, "connection write timeout")
 	mysqlQueryTimeout     = flag.Duration("mysql_server_query_timeout", 0, "mysql query timeout")
 
+	// TODO: enable this once the loadtest is done
+	// mysqlConnBufferPooling = flag.Bool("mysql_conn_buffer_polling", false, "If set, vtgate will pool connection buffers")
```

Comment:
nit for later: would rename flag to have consistent prefix with other flags in this file. something like `mysql_server_read_conn_pooling`? would also update flag description to specify that it's connection reading as connection writing is already pooled.